config.go

```diff
@@ -24,11 +24,13 @@ Full populated config:
 		"Port":			8080,
 		"Bind":			"127.0.0.1"
 	},
-	"Authenticator": {
-		"Type":			"ldap",
-		"LdapHost":		"domaincontroller.example.com",
+	"LDAP": {
```

Comment:
Rather keep "imqs" out of the examples. Use example.com and other names like it.